components/create_comment/index.js

```diff
@@ -3,14 +3,14 @@
 
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
 import {isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
 import {haveIChannelPermission} from 'mattermost-redux/selectors/entities/roles';
 import {getBool} from 'mattermost-redux/selectors/entities/preferences';
 import {getAllChannelStats, getChannelMemberCountsByGroup} from 'mattermost-redux/selectors/entities/channels';
 import {makeGetMessageInHistoryItem} from 'mattermost-redux/selectors/entities/posts';
 import {resetCreatePostRequest, resetHistoryIndex} from 'mattermost-redux/actions/posts';
-import {getChannelTimezones, getChannelMemberCountsByGroup as selectChannelMemberCountsByGroup} from 'mattermost-redux/actions/channels';
+import {getChannelTimezones, getChannelMemberCountsByGroup as loadChannelMemberCountsByGroup} from 'mattermost-redux/actions/channels';
```

Comment:
Actually, to keep this somewhat synchronized with mobile, it as suggested to me that keep this called as `getChannelMemberCountsByGroup` and import the selector above `as selectChannelMemberCountsByGroup`. 0/5 but just idea to keep it parallel with mobile code but not super important. I'll leave it up to cause currently it's good too.

Reply:
done @hahmadia